fvtest/compilertest/control/TestJit.cpp

```diff
@@ -161,7 +167,7 @@ initializeTestJit(TR_RuntimeHelper *helperIDs, void **helperAddresses, int32_t n
 
    initializeAllHelpers(jitConfig, helperIDs, helperAddresses, numHelpers);
 
-   if (commonJitInit(fe, options) < 0)
+   if (commonJitInit(fe, (char *)optionsWithVerifier.c_str()) < 0)
```

Comment:
Are you certain this is safe? `c_str` returns a `const char*`. This looks like an unsafe case. How are you certain that `commonJitInit` is not modifying the underlying array? At the very least I'd like to see a `const_cast` here.